packages/dds/tree/src/treeFactory.ts

```diff
@@ -24,20 +24,7 @@ import {
 	TreeFieldFromImplicitField,
 	TreeView,
 } from "./class-tree";
-
-/**
- * Configuration to specialize a Tree DDS for a particular use.
- * @internal
- */
-export interface TreeOptions extends SharedTreeOptions {
-	/**
-	 * Name appended to {@link @fluidframework/datastore-definitions#IChannelFactory."type"} to identify this factory configuration.
-	 * @privateRemarks
-	 * TODO: evaluate if this design is a good idea, or if "subtype" should be removed.
-	 * TODO: evaluate if schematize should be separated from DDS construction.
-	 */
-	readonly subtype?: string;
-}
```

Comment:
This is used to modify the channel type string, which Tony/Vlad were dubious about.  IIRC, they recommended preserving the creation options that effect the op/summary formats in IChannelAttributes.